app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewDialog.kt

```diff
@@ -173,26 +224,43 @@ class LinkPreviewDialog : ExtendedBottomSheetDialogFragment(), LinkPreviewErrorV
         showError(throwable)
     }
 
+    override fun onStart() {
+        super.onStart()
+        if (viewModel.fromPlaces) {
+            disableBackgroundDim()
+            BottomSheetBehavior.from(requireView().parent as View).state = BottomSheetBehavior.STATE_EXPANDED
```

Comment:
This makes the dialog very jumpy and glitchy when being shown. Can we remove this logic? (and tell Robin it can't work this way.) (maybe we could increase the default `bottomSheetPeekHeight` a little bit instead?)

Reply:
Sounds good to me. I checked if we put a higher peek height (e.g. `400dp`), it will still have the jumpy issue, and it looks like showing the gallery causes the issue. I'll let Robin know about the issue and remove it, thanks!